build.sbt

```diff
@@ -82,9 +87,89 @@ coffeelint := {
 
 (compile in Compile) := ((compile in Compile).dependsOn(yarnInstall)).value
 
+
+lazy val bundle = taskKey[Pipeline.Stage]("Bundle script files using Rollup")
+
+includeFilter in bundle := "*.js" || "*.js.map"
+excludeFilter in bundle := HiddenFileFilter
+
+bundle := Def.task {
+  val streamsValue = streams.value
+  val inputDir = webTarget.value / "rollup" / "sync-rollup" / "main"
+  val outputDir = webTarget.value / "rollup" / "main"
+  val scriptsDir = "javascripts" + java.io.File.separator
+  val include = (includeFilter in bundle).value
+  val exclude = (excludeFilter in bundle).value
+
+  (inputMappings: Seq[PathMapping]) =>
```

Comment:
Could you wrap everything from this line (104) to line 135 in curly braces?  I'm actually astonished that SBT would parse this correctly without them.  (*Is* it even parsing correctly, or might it be doing unintended nonsense?)

Reply:
Thank you for noticing that, my mind was probably in Coffeescript mode when I wrote that piece of code... Braces are of course added in the new commit.